code/src/index.js

```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 import { createRoot } from 'react-dom/client';
-import './index.css';
+import './styling/index.css';
+import './styling/singlemovie.css';
+import './styling/footer.css'
+import './styling/trendinglist.css'
+import './styling/notfound.css'
 import { App } from './App';
 
```

Comment:
Impressing and clear to follow and read when you have the CSS for different components in different files!